.buildkite/pipeline.yml

```
@@ -5,6 +5,8 @@ steps:
       echo "build:ci --host_platform=@rules_haskell//haskell/platforms:linux_x86_64_nixpkgs" > .bazelrc.local
       nix-shell --arg docTools false --pure --run '
       set -e
+      # Ensure that the Nixpkgs and bindists Bazel versions match.
```

what about

```suggestion
      # Ensure that the Nixpkgs bazel version matches with what's specified in .buildkite/bazel-version
```

?

---

Do we need to run the same test in `.buildkite/bindists-pipeline` too?

---

I mean, we don't have to, because we do the fetching there, so maybe update the comment to

```suggestion
      # Ensure that the Nixpkgs bazel version matches with what's specified in .buildkite/bazel-version and fetched in `.buildkite/bindists-pipeline` for the bindists version
```

---

The version is used to construct the URL from which Bazel is fetch, so checking the version against that file once again is redundant.

I've expanded the comment as suggested.